lib/ember-cli/app.rb

```diff
@@ -84,45 +82,34 @@ def wait
       MSG
     end
 
-    def ember_path
-      @ember_path ||= app_path.join("node_modules", ".bin", "ember").tap do |path|
-        fail <<-MSG.strip_heredoc unless path.executable?
-          No local ember executable found. You should run `npm install`
-          inside the #{name} app located at #{app_path}
-        MSG
+    def method_missing(method_name, *)
+      if path_method = supported_path_method(method_name)
+        paths.public_send(path_method)
+      else
+        super
```

Comment:
How do you feel about something like:

```ruby
module EmberCLI
  class App < SimpleDelegator
     def initialize(name, options={})
       @name, @options = name.to_s, options
       super(PathSet.new(self))
     end
  end
end
```

instead of dynamically delegating to the paths?